src/actions/functionStore.js

```diff
@@ -0,0 +1,75 @@
+//TODO: I don't like the filename... to many files with the same name
```

Comment:
I don't think the filename is bad, it makes sense with the folder tree being that these are actions against the function store. You could remove this comment unless you have a better idea...